tests/ports/webassembly/py_proxy_get.mjs

```diff
@@ -0,0 +1,13 @@
+// Test `<py-obj> get <attr>` on the JavaScript side, which tests PyProxy.get.
+
+const mp = await (await import(process.argv[2])).loadMicroPython();
+
+mp.runPython(`
+x = {"a": 1}
+`);
+
+const x = mp.globals.get("x");
+console.log(x.a === 1);
+console.log(x.b === undefined);
+console.log(typeof x[Symbol.iterator] === "function");
+console.log(Object.prototype.toString.call(x) === "[object Object]");
```

Comment:
Is there are simpler test to do here, maybe mirror `py_proxy_has.mjs`:
```js
console.log(x[Symbol.toStringTag]);
```
?

To me, that makes it easier to see exactly what is being tested here (namely, not-a-string as index).

Reply:
I am not sure a comment would help but the reason for that test verbosity is to be sure that if JS APIs implicitly access unknown symbols these are handled and `toString.call(ref)` is a classic example of such implicit access but you are right it's not clear although that `console.log` would return `null` which is confusing indeed to me